attributes/default.rb

```diff
@@ -1,29 +1,29 @@
-default[:ruby_installer][:method] = 'package'
-default[:ruby_installer][:package_removals] = []
-default[:ruby_installer][:package_name] = true
-default[:ruby_installer][:rubygem_package] = false
-default[:ruby_installer][:rubydev_package] = true
-default[:ruby_installer][:source_version] = '1.9.3-p392'
-default[:ruby_installer][:source_install_dir] = '/usr/local'
-default[:ruby_installer][:source_rubygems_version] = '1.8.24'
-default[:ruby_installer][:source_rubygems_force] = false
-default[:ruby_installer][:source_optimize] = false
-default[:ruby_installer][:source_optimization_level] = 1
-default[:ruby_installer][:source_falcon_patch] = false
-default[:ruby_installer][:source_package_dependencies] = case node.platform_family
+default['ruby_installer']['method'] = 'package'
+default['ruby_installer']['package_removals'] = []
+default['ruby_installer']['package_name'] = true
+default['ruby_installer']['rubygem_package'] = false
+default['ruby_installer']['rubydev_package'] = true
+default['ruby_installer']['source_version'] = '1.9.3-p392'
+default['ruby_installer']['source_install_dir'] = '/usr/local'
+default['ruby_installer']['source_rubygems_version'] = '1.8.24'
+default['ruby_installer']['source_rubygems_force'] = false
+default['ruby_installer']['source_optimize'] = false
+default['ruby_installer']['source_optimization_level'] = 1
+default['ruby_installer']['source_falcon_patch'] = false
+case node['platform_family']
```

Comment:
Alternatively, you could probably delegate this to the `build-essentials` cookbook.